server/src/main/java/com/cloud/user/AccountManagerImpl.java

```diff
@@ -2565,6 +2556,22 @@ private UserAccount getUserAccount(String username, String password, Long domain
         }
     }
 
+    protected void updateLoginAttemptsWhenIncorrectLoginAttemptsEnabled(UserAccount account, boolean updateIncorrectLoginCount,
+                                                                      int allowedLoginAttempts) {
+        int attemptsMade = account.getLoginAttempts() + 1;
+        if (allowedLoginAttempts > 0 && updateIncorrectLoginCount) {
```

Comment:
@nvazquez 
just a suggestion: check the opposite conditions to reduce indent ?

Reply:
Done, thanks